recipes/langchain-tavily/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "langchain-tavily" %}
+{% set version = "0.2.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/langchain_tavily-{{ version }}.tar.gz
+  sha256: ab4f5d0f7fcb276a3905aef2e38c21a334b6cbfc86b405a3238fdc9c6eae1290
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - poetry-core
+    - pip
+  run:
+    - python >= {{ python_min }}
```

Comment:
```suggestion
    - python >={{ python_min }}
```

Reply:
sorry for the early ping, i missed the lint errors at first 😄